index.html

```diff
@@ -671,10 +671,241 @@
 				<p class="ednote">Placeholder for inter-publication search.</p>
 			</section>
 
-			<section id="enh-pagination">
-				<h3>Pagination</h3>
+			<section id="Layout">
+				<h3>Layout</h3>
+
+				<p class="issue advisement">This section needs thorough review not only from the publishing community
+					to ensure that the proposed behavior is desirable,
+					but also from Web Browser vendors and other implementors of layout engines
+					to ensure that is it implementable.</p>
+
+				<p>The layout and rendering of <a>Web Publications</a> is governed by the usual rules that apply to all web content.
+				[[!HTML]] (including XHTML) documents are styled and laid out according to the rules of <a href="https://www.w3.org/TR/CSS/">CSS</a>,
+				[[!SVG]] files are rendered as normally expected of that format, and so on.
+				This specification requires no particular profile or subset of CSS, HTML, or SVG to be supported,
+				other than the expectations set for these technologies by their respective specifications.</p>
+
+				<div class=note>
+					<p>This specification intentionally refrains from introducing any new layout feature.
+					Any shortcoming of the web platform in terms of layout needs to be addressed for the whole web platform,
+					which means via CSS.</p>
+
+					<p>This working group recognizes that the features offered by CSS specifications have limitations,
+					and that in some cases implementations may be incomplete.
+					However, attempts to address these shortcomings for the exclusive purpose of <a>Web Publications</a>
+					would at best have no effect,
+					and at worst lead to a divergence between “ordinary” web content and <a>Web Publications</a>,
+					instead of the convergence this specification aims to achieve.</p>
+
+					<p>This working group will work with other relevant groups of the W3C to address
+					platform wide limitations that negatively impacts <a>Web Publications</a>,
+					and invites readers to do the same.</p>
+				</div>
+
+				<p>For the purposes of layout, each resource of a <a>Web Publication</a> is treated as a separate document.
+				User Agents must not mix content from multiple resources in the same rendering:
```

Comment:
This is not true.  Resources are not the same as documents...

Reply:
I am using the word document in the CSS / HTML sense. The thing that has a DOM, a document object, and that has a defined model for being styled. Not an abstract concept of document, which could/should be used to refer to the entire publication.

Any suggestion as to how to make that unambiguous?

Now, we may still disagree as to whether that's a good idea (as has been discussed in https://github.com/w3c/wpub/pull/65#issuecomment-330925058 and https://github.com/w3c/wpub/pull/65#issuecomment-331059874 and https://github.com/w3c/wpub/pull/65#issuecomment-331098264), that's a separate question.